### PR TITLE
[IMP] l10n_no: add standard tax code

### PR DESCRIPTION
**Pull request by proose**
For correct reporting of the SAFT report in Norway, a standard code for taxes is used. Until now, it was only found in the description or in the tax name (in earlier version). For more robustness, the code is now added as a field in the Norwegian localisation.

Linked upgrade PR: https://github.com/odoo/upgrade/pull/6742

task-3975794

### CODE REVIEW

File: addons/l10n_no/views/account_tax.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<odoo>
+    <record id="account_tax_form_inherit_l10n_no" model="ir.ui.view">
+        <field name="name">account.tax.form.inherit.l10n_mx</field>
```
[the remark this answers is not in the document]
Comment (proose) [reply addons/l10n_no/views/account_tax.xml:4]: WOOOPS

File: addons/l10n_no/views/account_tax.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+        <field name="model">account.tax</field>
+        <field name="inherit_id" ref="account.view_tax_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='analytic']" position="before">
```
[the remark this answers is not in the document]
Comment (proose) [reply addons/l10n_no/views/account_tax.xml:8]: Alright, I'll move it, I asked OLMA to check for any changes, I guess he'll tell us if that doesn't seem ok for him :D

File: addons/l10n_no/data/template/account.tax-no.csv [hunk 3]
```diff
@@ -1,141 +1,141 @@
...
-"","","","","","","","","","tax","invoice","","","","",""
-"","","","","","","","","","tax","invoice","chart2705","-100","+92 Tax","",""
-"","","","","","","","","","base","refund","","","-92 Base","",""
-"","","","","","","","","","tax","refund","","","","",""
-"","","","","","","","","","tax","refund","chart2705","-100","-92 Tax","",""
+"id","name","sequence","description","invoice_label","amount","amount_type","type_tax_use","tax_group_id","l10n_no_standard_code","repartition_line_ids/repartition_type","repartition_line_ids/document_type","repartition_line_ids/account_id","repartition_line_ids/factor_percent","repartition_line_ids/tag_ids","description@nb_NO","invoice_label@nb_NO"
+"tax2","25%","0","1 Input VAT high rate 25%","Deduction for incoming VAT","25.0","percent","purchase","tax_group_25","1","base","invoice","","","","1 Inngående mva høy sats 25%","Fradrag for inngående mva"
```
[the remark this answers is not in the document]
Comment (proose) [reply addons/l10n_no/data/template/account.tax-no.csv:2]: I asked him on the task, yeah

**Comment by proose** [comment]
@gawa-odoo Here is the upgrade PR: https://github.com/odoo/upgrade/pull/6742, could you take a look at it ? (still waiting for PO to check everything)
Could I ask for you hindsight 
Either the upgrade script uses the description to set up the standard tax codes (what I did): that allows for user-added taxes to have a STC (standard tax code), if the user respected the rule to add the STC in the tax description.
Either I just fetch the taxes by their ref, and manually put the correct STC: the upgrade script would only work on data taxes but avoid user errors.
What would you prefer ?

File: addons/l10n_no/views/account_tax.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (proose) [reply addons/l10n_no/views/account_tax.xml]: He didn't seem bothered by the current field position :+1:

**Comment by gawa-odoo** [comment]
@robodoo r+

**Pull request closed without merging**